data/cameras.xml

```diff
@@ -8644,6 +8644,28 @@
 		<Crop x="0" y="0" width="-938" height="-4"/>
 		<Sensor black="65" white="16383"/>
 	</Camera>
+        <Camera make="FUJIFILM" model="GFX50S II">
```

Comment:
These aren't tabs.

Reply:
Sorry - copy/paste from MSYS2's mintty changes those then, I'll keep it in mind.